### DISCUSSION TITLE
Timezone docker and quartz miss match causing issues

### DISCUSSION THREAD

**Discussion by thedangler** [quartznet/quartznet#1704] [Q&A] [answered]
Hello the docs say that it will use the timezone of the server time however that does not seem to be the case.
The times saved in quartz are all UTC time where our server timezone is set to America/Toronto. 

For example this line:
trigger.StartAt((DateTimeOffset)_utilityViewModel.StartTime); 
should of saved it for roughly  2022-08-11 10:21 AM but in quartz its saved as 2022-08-11 14:18:01.000.

_utilityViewModel.StartTime when debugging is my local time.

Event when dealing with times and using ToLocalTime() its not using America/Toronto timezone.

Wondering how I can fix this 

Thank you

**Comment by lahma** [comment, marked as answer]
You should give always UTC times to Quartz API as input, or will then use trigger's time zone or local if trigger has non defined.

**Reply by thedangler** [reply]
Ok but when getting it out how come ToLocalTime() doesn't change it to the local timezone?

**Reply by lahma** [reply]
Create a repro solution and might be easier to understand.
